lib/ramble/ramble/test/experiment_set.py

```diff
@@ -409,8 +409,100 @@ def test_experiment_names_match(mutable_mock_workspace_path):
         exp_set.set_workload_context(wl_name, wl_vars, None)
         exp_set.set_experiment_context(exp_name, exp_vars, None, exp_matrices)
 
-        assert 'series1_4_2' in exp_set.experiments.keys()
-        assert 'series1_12_4' in exp_set.experiments.keys()
+        assert 'basic.test_wl.series1_4_2' in exp_set.experiments.keys()
+        assert 'basic.test_wl.series1_12_4' in exp_set.experiments.keys()
 
         for exp, app in exp_set.all_experiments():
-            assert exp == app.variables['experiment_name']
+            assert exp == app.expander.expand_var('{experiment_namespace}')
+
+
+def test_cross_experiment_variable_references(mutable_mock_workspace_path):
+    workspace('create', 'test')
```

Comment:
Now we can change the workspace dir I wonder if we should be more explicit about putting these test workspaces somewhere more temporary ?